src/webgpu/api/validation/capability_checks/features/pipeline_statistics_query.spec.ts

```diff
@@ -0,0 +1,40 @@
+export const description = '';
+
+import { params, pbool, poptions } from '../../../../../common/framework/params_builder.js';
+import { makeTestGroup } from '../../../../../common/framework/test_group.js';
+import { ValidationTest } from '../../validation_test.js';
+
+export const g = makeTestGroup(ValidationTest);
+
+g.test('create_query_set')
```

Comment:
nit:
```suggestion
g.test('createQuerySet')
```
since it's the name of the function being tested.